safe/processors/population_post_processors.py

```diff
@@ -74,6 +74,9 @@
     'description': tr(
         'A post processor to add the population displacement ratio according '
         'to the hazard class'),
+    'run_filter': {
+        'exposure': ['population']
```

Comment:
Is it possible to use the get the value from the dictionary, (e.g. `'exposure': [population['key']]`) so that it's easier to maintain.

Reply:
@ismailsunni do you mean 'exposure': [exposure_population['key']] ?

Reply:
@mbernasocchi yes.

Reply:
I think you missed this one @mbernasocchi